src/Init/Tactics.lean

```diff
@@ -207,6 +207,28 @@ the first matching constructor, or else fails.
 -/
 syntax (name := constructor) "constructor" : tactic
 
+/--
+Applies the second constructor when
+the goal is an inductive type with exactly two constructors, or fails otherwise.
```

Comment:
This is also some odd word-wrapping.

[Rewrap](https://open-vsx.org/extension/stkb/rewrap) is a good way to make it easy to fix these.

Reply:
Oh. It was my attempt at semantic line-wrapping.